yast2_lan_restart-yast2_lan_global_options_tab-leap-15-20180124.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "ypos": 144,
+      "xpos": 85,
+      "type": "match",
+      "width": 137,
+      "height": 29
+    }
+  ],
+  "tags": [
+    "yast2_lan_global_options_tab"

```

Comment:
I guess if this is leap specific, let's add ENV-LEAP-1 to all needles here, or to none if we expect them to work on TW, for example.

Reply:
yes, these are leap specific needles. Added.